INTHEWILD.md

```diff
@@ -343,6 +343,7 @@ Currently, **officially** using Airflow:
 1. [Syapse](https://www.syapse.com/) [[@zedmor](https://github.com/zedmor)]
 1. [T2 Systems](http://t2systems.com) [[@unclaimedpants](https://github.com/unclaimedpants)]
 1. [TEK](https://www.tek.fi/en) [[@telac](https://github.com/telac)]
+1. [The Dyrt](https://thedyrt.com/) [[@mfjackson](https://github.com/mfjackson)]
```

Comment:
This should be moved after:

  1. [The Climate Corporation](https://climate.com/) [[@jmelching](https://github.com/jmelching)]